README.md

```diff
@@ -1,65 +1,71 @@
+The course structure is outlined in the <a target="_blank" rel="noopener noreferrer" href="https://shawnrhoads.github.io/gu-psyc-347/module-00-00_Syllabus.html">syllabus</a> and <a target="_blank" rel="noopener noreferrer" href="https://shawnrhoads.github.io/gu-psyc-347/module-00-01_Course-Schedule.html">schedule</a>. These include course goals and student expectations, an example timeline of the course (i.e., how long to spend on each module), prerequisites and required materials/software, and example assessments of learning (e.g., assignments, exercises, readings, projects).
+
+All course materials are also designed to be modular. For example, if students are knowledgeable in Python basics and Jupyter Notebook (**Module 01**), then instructors could begin with model fitting (**Module 02**). If students are knowledgeable in model fitting with maximum likelihood estimation (**Module 02**), instructors could start with reinforcement learning (**Module 03**). Instructors can adopt, remix, transform, and build upon these materials for their own course or for other educational purposes (e.g., workshops, single class sessions) under a <a rel="license" target="_blank" rel="noopener noreferrer" href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.
+
+For additional background on the content covered in each tutorial (technical or theoretical), instructors are encouraged to review the references and/or resources listed at the beginning of each tutorial. In addition, instructors should familarize themselves with the <a target="_blank" rel="noopener noreferrer" href="https://shawnrhoads.github.io/gu-psyc-347/module-00-03_Reading-List.html">recommended readings</a> associated with each module and use them to introduce students to each topic, generate discussions and ideas among students, and provide relevant background for each tutorial. 
 
-## Contributing 🙋‍♀️
-Please visit <a target="_blank" rel="noopener noreferrer" href="https://shawnrhoads.github.io/gu-psyc-347/module-00-06_Contributing.html">this page</a> if you would like to help improve and/or expand the content in this Jupyter Book!
+## For Contributors 🤝
+
+While the course aims to incorporate a diverse curriculum to introduce students to the basics of modeling in the context of social behavior, there is still so much still left out (and still much unknown in this growing field). That being said, other researchers and educators are invited to help improve and expand the content included in this Jupyter Book!
```

Comment:
"there is still so much still left out (and still much unknown in this growing field)": a lot of "still"s!

Reply:
good catch!